framework/log/SyslogTarget.php

```diff
@@ -26,6 +26,10 @@ class SyslogTarget extends Target
      * @var int syslog facility.
      */
     public $facility = LOG_USER;
+    /**
+     * @var int openlog option
+     */
+    public $option = LOG_ODELAY | LOG_PID;
```

Comment:
I would prefer the name `$options` as these are multiple options that can be combined.